sematic/config/config.py

```diff
@@ -16,7 +16,8 @@
 logger = logging.getLogger(__name__)
 
 # Support for dropping columns added in 3.35.0
-MIN_SQLITE_VERSION = (3, 35, 0)
+# Support for JSON querying added in 3.38.0
```

Comment:
Not all users will have a sqlite version that meets this criteria. What is the failure mode for them?

Reply:
At this time, we only display a warning message.

https://github.com/sematic-ai/sematic/blob/3f6c785d26c1a833e5b0ea59551af2ff8cab3548/sematic/config/config.py#L26-L32

I believe writing metrics would fail without this version.
What we could do is catch the error in `SQLMetricsStorage` and display an error message in logs.